usl_pipeline/usl_lib/tests/storage/test_metastore.py

```diff
@@ -91,6 +91,38 @@ def test_study_area_as_header():
     )
 
 
+def test_study_area_delete_if_exists_skipped():
```

Comment:
I think the tests against a local firestore in 
[usl_lib/tests/integration/test_metastore_integration.py](https://github.com/UrbanSystemsLab/climateiq-cnn/blob/4156d3be61f86caf06934e6c7d28e70481bcdce6/usl_pipeline/usl_lib/tests/integration/test_metastore_integration.py)
are generally more useful. They'd be a good place to test out the removal of subcollections I mention in another comment.

Reply:
Done!